native/src/main/java/io/ayash/blockingqueue/ModuleUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
No need for WSO2 license header here as this is your code.

Reply:
Check this in other places as well